js/views/modals/purchase/Purchase.js

```diff
@@ -452,6 +452,7 @@ export default class extends BaseModal {
         singleSelect: true,
         selectFirst: true,
         radioStyle: true,
+        useCache: false,
```

Comment:
well then:

1.) Do we need a larger discussion on this. @cpacia was under the impression that we would opt for cached mods to decrease latency.
2.) @jjeffryes If we are going to go with the uncached option, then how is this PR necessary as that is what's in place now? If you don't pass in a value for usecache to the moderators View, it is undefined and then cast into a boolean... so it would be sent as false.